### ISSUE TITLE
baby cry dataset and test accuracy

### ISSUE THREAD

**Issue by happyprotean** [giulbia/baby_cry_detection#23]
Hi sir,  I use your model to detect baby cry and the test accuracy and F1 score are both around 0.95. But the question is the performance is too great. I come up two reasons to explain this. First, the test dataset is too simple. Second, I notice that some of the data examples you provided come from the same audio which means long aduio is cut to several 5 seconds clips. Maybe this contributes to the test performance.
And I am working on detect baby cry recently but find there is no public big dataset for this goal. I collect some data here:ESC-50, donateacry,DCASE-2017 rare event detection and this respository. Do you know some other baby cry dataset?  It would be much appreciated if you let me know.

**Comment by liupeng678** [comment]
SPLANN  is a project for baby crying .but I don't  how to access it.I come from guangzhou

**Comment by paolo626** [comment]
> Hi sir, I use your model to detect baby cry and the test accuracy and F1 score are both around 0.95. But the question is the performance is too great. I come up two reasons to explain this. First, the test dataset is too simple. Second, I notice that some of the data examples you provided come from the same audio which means long aduio is cut to several 5 seconds clips. Maybe this contributes to the test performance.
> And I am working on detect baby cry recently but find there is no public big dataset for this goal. I collect some data here:ESC-50, donateacry,DCASE-2017 rare event detection and this respository. Do you know some other baby cry dataset? It would be much appreciated if you let me know.

SPLANN is a project for baby crying .but I don't how to access it.I come from guangzhou

**Comment by happyprotean** [comment]
@liupeng678 Yes, I have look into the SPLANN project page but found no access to this dataset. What a pity!!! Do you also work on baby crying detection?

**Comment by paolo626** [comment]
> @liupeng678 Yes, I have look into the SPLANN project page but found no access to this dataset. What a pity!!! Do you also work on baby crying detection?

My previous work was computer vision, and I began to learn baby crying recently. Can you give me a wechat Id  or QQ? Maybe we can do it more qucikly.

**Comment by happyprotean** [comment]
@liupeng678 Hi you can add my wechat: fhx456321

**Comment by cuong3004** [comment]
Have you got SPLANN, can I have the download link, I am also conducting a research related to baby cry

**Comment by dcleres** [comment]
@cuong3004 did you manage to get the link? I would also be interested. Thank you very much!

**Comment by mThanh1311** [comment]
Hello there, I am also conducting a research related to baby cry so i need a data for this project. Any have SPLANN for this.

**Comment by elianedl** [comment]
Hello, can I have too the download link, I am also conducting a project related to baby cry.

**Comment by baselebido** [comment]
Hello there, can I have the download link please, I am also conducting a project related to baby crying.

**Comment by ayanasherin** [comment]
Hello, I am also doing a project related to baby cry, but there are not a lot of datasets available for this. I could only find donateacry. Can anyone help me with finding another dataset

**Comment by dalia1112** [comment]
Hello , can I have the download link please, I am also conducting a project related to baby cry.

**Comment by niloofarjazaeri** [comment]
Hello, as a grad student working on the infant cry project, I also appreciate the link of access.

**Comment by hadeelayman8** [comment]
Hello, I am currently working on my graduation project and it is about infant cry, I appreciate the access to the data.

**Comment by Niketiiitd** [comment]
As a undergrad researcher, can I have dataset link ?
